pkgs/tools/misc/lice/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, python3Packages }:
+
+python3Packages.buildPythonPackage rec {
+
+  package = "lice";
+  version = "0.4";
+  name = "${package}-${version}";
```

Comment:
Nitpick: Why not name = "lice-${version}". You don't use `package` anywhere else.

Reply:
cargo-culted from a default.nix for personal project.  Will be fixed.